platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -5496,6 +5496,10 @@ void DisplayServerX11::window_start_drag(WindowID p_window) {
 	ERR_FAIL_COND(!windows.has(p_window));
 	WindowData &wd = windows[p_window];
 
+	if (wd.embed_parent) {
+		return; // Embedded window.
+	}
```

Comment:
I have tested the following:
```suggestion
	Window target = wd.embed_parent ? wd.embed_parent : wd.x11_window;
```

I also replaced `wd.x11_window` by `target` in the two locations below.

This approach works for me on linux and allows resizing the embedder window

Reply:
Should work, but I'm not sure if dragging editor embedded window should be supported, it will have editor frame anyway and other window movement methods (like `window_set_position`/`window_set_screen`) are fully disabled.